setup.cfg

```diff
@@ -0,0 +1,19 @@
+[yapf]
+based_on_style = google
+indent_width = 2
+column_limit = 90
```

Comment:
I believe this is a divergence from TensorFlow's `.pylintrc`, which we're currently using!  that has a conservative `column_limit = 79`.

if the change is intentional, it should be at least be better documented in the commit history and/or top-level for this PR. personally I like the character limit at 80, since it's _just right_ for viewing full scripts side-by-side on pretty much any modern screen

Reply:
put it back to 80, let's go with that 👍